docs_src/microservices/configuration/Ch-Configuration.md

```diff
@@ -254,53 +262,59 @@ same port as the HTTP API. By default this is
 
 ### Running on Docker
 
-For ease of use to install and update, the microservices of EdgeX
-Foundry are also published as Docker images onto Docker Hub, including
+For ease of use to install and update, the micro services of EdgeX
+Foundry are published as Docker images onto Docker Hub, including
 Registry:
 
 > <https://hub.docker.com/r/edgexfoundry/docker-core-consul/>
 
 After the Docker engine is ready, users can download the latest Consul
 image by the docker pull command:
 
-> docker pull edgexfoundry/docker-core-consul
+``` bash
+docker pull edgexfoundry/docker-core-consul
+```
 
 Then, startup Consul using Docker container by the Docker run command:
 
-> docker run -p 8400:8400 -p 8500:8500 -p 8600:8600 \--name
-> edgex-core-consul \--hostname edgex-core-consul -d
-> edgexfoundry/docker-core-consul
+``` bash
```

Comment:
Why not just download compose file and run

```
docker-compose -f <file name> up consul
```

Add skip the rest of these steps?

Reply:
Was in the original doc @lenny-intel .  The whole section at the bottom about getting and running Consul seemed out of place to me.  I would prefer to leave the whole section out.  Thoughts?

Reply:
Agreed. Does seem out of place. KILL IT! ;-)